### PR TITLE
Project initialize

### PR DESCRIPTION
**Pull request by MenendezON**
This Pull Request represents that I have taken the comments from a closed PR into account.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,245 @@
...
+
+
+**Microverse First Project** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
```
Comment (nahnahdev) [inline README.md]: - After referring to this [PR](https://github.com/MenendezON/Hello-Microverse/pull/1)
- [x] Great job so far. As the previous reviewer mentioned,  All lines beginning with` >` just like this line throughout your README Should be deleted. These lines only serve as a guide on how you should fill particular sections of this document and should be deleted once all sections are filled in. 👍🏾
- [x] Also I noticed that you closed the original pull request and submitted a new one. However, here in Microverse, you should use the same pull request to request for review until the Pull request is approved by doing so the next reviewer will have access to the comments of the reviewer before and check if you implemented the suggested changes. Also, you could have reopened the old PR and submitted it so next time kindly ensure that you use one PR :100:  You can open this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md) to learn more :+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,245 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
```
Comment (nahnahdev) [inline README.md]: - As the previous Code Reviewer mentioned, Please delete all comments from your code base. Comments make the repo untidy. :smile:

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @MenendezON , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
- Good use of GitHub flow :heavy_check_mark: 
- Project well setup :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] I noticed that you didn't install [HTML/CSS linters](https://github.com/microverseinc/linters-config/tree/master/html-css) in your project. Please note that this is an essential part of your repository setup and even though this is an optional suggestion, I will advise that you include it in your project so that you will practice how to set up linters and how to solve linter errors :100:  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (MenendezON) [reply README.md]: I don't know what happened, I didn't close the PR

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @MenendezON    ,
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## To Highlight :100:
- No linters errors :heavy_check_mark:
- Descriptive commit messages :heavy_check_mark:
- clean code :heavy_check_mark:

## Required Changes :recycle:



- [x] kindly consider customising your readme so that its related to your current project  check on the below images 

Screen Shot 2023-03-23 at 16 17 48
Screen Shot 2023-03-23 at 16 18 26

_Check the comments under the review._
## Optional suggestions














**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Review by LionRouge1 — APPROVED** [review]
Hi @MenendezON  , 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Please add a summary describing in more detail the changes introduced. This will help other developers know what to expect from your project.
This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change.

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**